src/main/java/appeng/core/features/registries/WirelessRangeResult.java

```diff
@@ -25,8 +25,8 @@
 public class WirelessRangeResult
 {
 
-	public final float dist;
-	public final TileEntity te;
+	private final float dist;
+	private final TileEntity te;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "te" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)